multi-node/aws/pkg/config/templates/cluster.yaml

```diff
@@ -55,6 +55,9 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 # IP Address for controller in Kubernetes subnet
 # controllerIP: 10.0.0.50
 
+# Comma seperated list of Etcd endpoints defaults to http://10.0.0.50:2379 
+# etcdEndpoints: "http://10.0.0.50:2379"
```

Comment:
Can we put a note here that the "default etcd configuration" requires that etcdEndpoints = `http://<controllerIP>:2379`? In the event the controllerIP is changed to non-default, I want the need to change `etcdEndpoints` to be obvious.